composer.json

```diff
@@ -19,9 +19,9 @@
 
     "require": {
         "php":                      ">=5.3.2",
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "symfony/finder":           "~2.3|~3.0",
-        "symfony/validator":        "~2.3|~3.0",
+        "symfony/framework-bundle": "~2.3|~3.1",
```

Comment:
And Upgrade from 3.0 to 3.1 is not so difficult... https://github.com/symfony/symfony/blob/master/UPGRADE-3.1.md